CHANGELOG.rst

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ------------------
 
 * Fix login password field
+* Enable sourcemaps on gulpfile sass task
```

Comment:
would adapt it like "Enable sourcemaps generation when using ``--debug`` param on Gulp"